packages/cli/src/prompts/prompt.ts

```diff
@@ -24,6 +24,7 @@ interface InquirerQuestion {
   default?: any;
   choices?: ChoicesT;
   when?: ((answers: { [key: string]: any }) => boolean);
+  transformer?: ((value: string, answers: { [key: string]: any }) => string);
```

Comment:
What's the difference between this one and normalize?

Reply:
`normalize` is a custom function defined by us to specify the format of the answer, while `transformer` is an `inquirer.js` function that  returns a transformed value to display in the console. The latter only impacts what is shown while editing and it doesn't modify the answer. You can check the gif above, and see how the `ether`, `gwei`, etc is shown while typing an amount value.